ambf_utilities/tf_utils/scripts/frame.py

```diff
@@ -0,0 +1,200 @@
+""" This class will generate a Frame in 3D space to mimic Frame in pyKDL"""
+
+import numpy as np
+from numpy import pi, sin, cos, tan, arctan2
+from vector import Vector
+from twist import Twist
+from wrench import Wrench
+from rotation import Rotation
+from warnings import warn
+import sys
+
+
+class Frame(object):
+
+  M = Rotation()
```

Comment:
Any reason for these two attributes being declared as static? Wouldn't it be better to make them instance attributes?